src/components/BatchMint/sample.json

```
@@ -0,0 +1,4 @@
+{
+  "fields": ["address", "tokenURI"],
```

`address`→`to`だと思います！

リクエストボディ: https://docs.hokusai.app/docs/hokusai/b3A6NDQyMjg0MTY-mints-new-nft

上記ドキュメントだと`tokenURI`ではなく`tokenUri`となっていますが、`tokenURI`が正しいのでこちらは大丈夫です！(ドキュメントの修正お願いしておきます)

```suggestion
  "fields": ["to", "tokenURI"],
```